lib/src/rules/prefer_const_literals_to_create_immutables.dart

```diff
@@ -113,7 +118,8 @@ class _Visitor extends SimpleAstVisitor<void> {
         (node is ParenthesizedExpression ||
             node is ArgumentList ||
             node is ListLiteral ||
-            node is MapLiteral ||
+            node is SetOrMapLiteral ||
+            // todo (pq): check if this needs updating with the SetOrMapLiteral unification
```

Comment:
No. We still use MapLiteralEntry to represent a `key : value` structure.

Reply:
Thanks!